runtime/j9vm/java11vmi.c

```diff
@@ -342,13 +343,14 @@ addPackageDefinition(J9VMThread * currentThread, J9Module * fromModule, const ch
 	J9Package * j9package = createPackage(currentThread, fromModule, package);
 
 	if (NULL != j9package) {
+		Trc_MODULE_invokeHashTableAtPut(currentThread, "addPackageDefinition", classLoader->packageHashTable, &j9package, j9package, "true");
```

Comment:
Is the packageHashTable the relevant thing to log here?  I think the classloader itself might be more useful as it's easier to correlate to other debug data and gives the right `!j9classloader 0xaddress` to find the loader in core dump

Reply:
Agreed, will change to logging `classLoader` instead. (This is the part inherited from existing trace points.)